back-end/db.js

```diff
@@ -1,4 +1,5 @@
 const mysql = require('mysql2');
+require('dotenv').config();
```

Comment:
You do not need this over here, in the server it must be before the DB file require